-- 使用error函数抛出一个异常
-- error({code = 10001, msg = "发生错误了！"}) 

--print("test1")

--require "log"
   error("你的智商不多了，赶紧休息，恢复一下吧");

str = "I am so cool"  
tbl = {name = "shun", id = 20114442}  
function add(a,b)  
    return a + b  
end


function run(a,b)  
    d2(a);
    d2(b);
end